packages/desktop/bin/dump_artifacts.ts

```diff
@@ -154,22 +154,23 @@ async function installPackages(
     );
     requirements.push(streamlitWheel);
   } else {
-    const packageJson = require(path.resolve(__dirname, "../package.json"));
-    const version = packageJson.version;
-
-    const jsDelivrFilesUrl = `https://data.jsdelivr.com/v1/package/npm/@stlite/kernel@${version}/flat`;
-    const jsDelivrFilesRes = await fetch(jsDelivrFilesUrl);
-    const jsDelivrFilesJson = await jsDelivrFilesRes.json();
-    const wheelFiles = jsDelivrFilesJson.files.filter((fileData) =>
-      fileData.name.endsWith(".whl")
-    );
-    const wheelUrls = wheelFiles.map(
-      (wheelFile) =>
-        `https://cdn.jsdelivr.net/npm/@stlite/kernel@${version}${wheelFile.name}`
-    );
+    const wheelsDir = path.join(__dirname, "../wheels");
+    const wheelFilePaths = await fsPromises
```

Comment:
Should this wheel file list be verified?
A malicious package can put a malicious wheel file to the `wheels` directory so it's loaded into the packaged app?

Reply:
However, if such an attack is possible, the attacker can edit any other parts of this package, so such verification is not possible anyway? Even the verifier code can be tricked.

Reply:
Change the code to hard-code the file names as a minimum defense, knowing it can't guard such attacks ultimately.